src/style.css

```diff
@@ -2,0 +1,70 @@
+@font-face {
+  font-family: Avenir;
+  src: url(/src/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(/src/fonts/Avenir-Heavy.ttf);
+  font-weight: 600;
```

Comment:
Similarly, verify that the font-weight specified here matches the actual weight of the 'Avenir-Heavy.ttf' font file. Incorrect font-weight can lead to inconsistent text appearance.